pennylane/beta/devices/default_tensor.py

```diff
@@ -153,7 +153,7 @@ class DefaultTensor(Device):
 
     _zero_state = np.array([1.0, 0.0], dtype=C_DTYPE)
 
-    def __init__(self, wires, shots=1000, representation="exact", contraction_method="auto"):
+    def __init__(self, wires, shots=None, representation="exact", contraction_method="auto"):
```

Comment:
This device previously was always set to analytic mode, and `shots=1000` was only used for the `sample` method. All other return types only support analytic mode.